UnitTest/tests/test_SD_RSL-definition.txt

```diff
@@ -5,7 +5,7 @@ attr RSL_411B11_0_0 room SD_RSL;;
 setstate RSL_411B11_0_0 off;;
 setstate RSL_411B11_0_0 2018-06-14 18:24:38 state off;;
 
-defmod test_rsl UnitTest RSL_411B11_0_0 ( 
+defmod test_SD_RSL UnitTest RSL_411B11_0_0 ( 
```

Comment:
Der Test muss nicht mehr in diesem Repo laufen, da ich das Modul prinzipiell ausgelagert habe.
Leider ist mein Patch Vorschlag im Sande verlaufen, so dass wir es nicht einfach referenzieren können :(